### ISSUE TITLE
Migrate Windows Builds to New Platform

### ISSUE THREAD

**Issue by nikhiljha** [VSCodium/vscodium#68]
The last version with a build for Windows was 1.29. Both 1.30 and 1.30.1 don't have Windows packages.

**Comment by stripedpajamas** [comment]
@nikhiljha ok so we should have Windows binaries for the latest release now. I'd like to leave this issue open as we work out getting the scheduled builds working.

**Comment by stripedpajamas** [comment]
@pavanagrawal123 what kind of webhook do you think would work in this case? Perhaps we could trigger something from Travis, where the scheduling works properly.

**Comment by pavanagrawal123** [comment]
@stripedpajamas that's what i was thinking hook over from Travis to Azure DevOps to trigger a "build"

**Comment by stripedpajamas** [comment]
@pavanagrawal123 I do not see a way to trigger builds via web request in the documentation... can you point me in the right direction?

**Comment by pavanagrawal123** [comment]
@stripedpajamas I haven't really used it before either, but I'll look into it ASAP!

**Comment by pavanagrawal123** [comment]
@stripedpajamas just looked. I don't see anything that indicates that we can trigger a build using a web request... which is quite disappointing... As an alternative, we could make Travis commit something to a branch and then use branch filters to pick that up via CI, but at that point, it might just be a better idea to run w/ AppVeyor....

**Comment by stripedpajamas** [comment]
Ok. I can run the builds manually for now, and I will change this issue to reflect the new task of migrating Windows builds over to AppVeyor or Travis (still waiting on secret support in Travis).

**Comment by jingyu9575** [comment]
@stripedpajamas [this](https://stackoverflow.com/a/52724205) seems to be the API for Azure Pipeline. Not sure if it is affected by the dormant status or not.

```
POST:

https://account.visualstudio.com/project/_apis/build/builds?api-version=4.1

Body

{ 
        "definition": {
            "id": number
        } 
}
```

**Comment by pavanagrawal123** [comment]
@jingyu9575 I just checked, this seems to be a perfect candidate to use.

Sorry I missed this @stripedpajamas, we should be able to query from Travis to use this.

**Comment by stripedpajamas** [comment]
Good find @jingyu9575 ! Thank you for that. I put in a PR that triggers the Azure build from Travis if a build is necessary. We will see if it works even if the account is in dormant state... will probably take a few weeks before we know. If not, we can reopen this issue.
